packages/cardano-graphql/src/operations/rewardsHistory.graphql

```diff
@@ -0,0 +1,37 @@
+# Review: this seems to be supported by dgraph. It would filter objects in graphql,
```

Comment:
This looks promising, but it's an optimisation right? If so, we can easily chop this out to a separate task